gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -129,6 +130,12 @@ def network_function_device_notification(self, context, notification_data):
                     event_id = self.rpc_event_mapping[resource][2]
                 break
 
+        nf_id = request_info.pop('nf_id')
+        nfi_id = request_info.pop('nfi_id')
+        nfd_id = request_info.pop('nfd_id')
+        request_info['network_function_id'] = nf_id
```

Comment:
Looks like this is done to avoid more code changes. but doesn't look good for submission.